docs/segments/manage_segments.rst

```diff
@@ -3,4 +3,493 @@
+
+.. image:: images/date-filters.png
+    :alt: Screenshot showing DatePicker.
+
+However, you can specify much more here. Mautic recognizes relative formats too (these string aren't translatable):
+
+* ``+1 day`` (you can also use ``1 day``)
+* ``-2 days`` (you can also use ``2 days ago``)
+* ``+1 week`` / ``-2 weeks`` / ``3 weeks ago``
+* ``+5 months`` / ``-6 months`` / ``7months ago``
+* ``+1 year`` / ``-2 years`` / ``3 years ago``
+
+Example (Consider that today is ``2022-03-05``):
+
+* ``Date identified equals -1 week`` returns all Contacts identified on 2022-02-26.
+* ``Date identified less than -1 week`` returns all Contacts identified before 2022-02-26.
+* ``Date identified equals -1 months`` returns all Contacts identified on 2022-02-05.
+* ``Date identified greater or equal -1`` year returns all Contacts identified 2021-03-05 and after.
+* ``Date identified greater than -1`` year returns all Contacts identified after 2021-03-05.
+  
+Beside this you can specify your date with text. These formulas are **translatable**, so make sure you use them in correct format.
+
+* ``birthday`` / ``anniversary``
+* ``birthday -7 days`` / ``anniversary -7 days``
+* ``today`` / ``tomorrow`` / ``yesterday``
+* ``this week`` / ``last week`` / ``next week``
+* ``this month`` / ``last month`` / ``next month``
+* ``this year`` / ``last year`` / ``next year``
+
+Example (Consider that today is ``2022-03-05``):
+
+* ``Date identified equals last week`` returns all Contacts identified between 2022-03-01 and 2022-03-07 (Monday - Sunday).
+* ``Date identified less than last week`` returns all Contacts identified before 2022-02-22.
+* ``Date identified equals last month`` returns all Contacts identified between 2022-02-01 and 2022-02-28.
+* ``Date identified greater or equal last year`` returns all Contacts identified 2021-01-01 and after.
+* ``Date identified greater than last year`` returns all Contacts identified after 2021-12-31.
+* ``Custom Contact date field equal birthday -1 day`` returns all Contacts identified every year on 03-04 (4th march).
+* ``Custom Contact date field equal anniversary -1 month`` returns all Contacts identified every year on 02-04 (4th february)
+
+Once you have created your Segment, any applicable Contact automatically add through the execution of a cron job. This is the essence of Segments.
+
+To keep the Segments current, create a cron job that executes the following command at the desired interval:
+
+``php /path/to/mautic/bin/console mautic:Segments:update``
+
+Through the execution of that command, it adds Contacts that match the filters and removes Contacts that no longer match. Any Contacts that were manually added will remain part of the list regardless of filters.
+
+.. vale off
+
+Delete all Contacts in a Segment
+********************************
+
+.. vale on
+
+Filter the Contacts in the Segment. The batch delete action in the Contact table allows deletion of up to 100 Contacts at one time. 
```

Comment:
What is the purpose of having the 'Filter the Contacts in a Segment' here?

If it is an instruction (eg that they need to first create the segment) we should be clearer here.

Maybe:

'It is possible to delete all contacts in a segment manually rather than with a Campaign action. To do this, first create a segment with filters that will pick up all the Contacts you want to delete.'